lib/pbench/server/api/resources/intake_base.py

```diff
@@ -100,24 +141,29 @@ def process_metadata(metas: list[str]) -> JSONOBJECT:
         """
         metadata: dict[str, Any] = {}
         errors = []
+        expanded = []
         for kw in metas:
+            expanded.extend(Term(term=kw, delimiter=",").parse_list())
```

Comment:
Perhaps,
```python
        expanded = [Term(term=kw, delimiter=",").parse_list() for kw in metas]
```

Reply:
That, I think, would generate a list of lists, which isn't what I want.

Reply:
Hmmm.  I'm sure I could solve that problem, but not without overflowing the line, and then Black would make that unpleasant...so, nevermind.

Reply:
Yeah, I thought about it for a moment and decided it wasn't worth the sparks...

Reply:
I think my problem is that I don't like `extend()` -- I much prefer `+=`.

(But, this is something I just need to get over -- I'm not requesting that change here...unless you _want_ to....  😉)

Reply:
Yeah, I usually think in terms of the *methods* in Python instead of thinking in terms of operator overloading. While I've always loved playing with OO languages that support operator overloads, it's also really easy for the code to become unclear. One could argue that's less of a problem with built-in classes, but I don't think most people have all the overloaded operators for `list` and `set` memorized with 100% certainty: I know I don't.